search/search_param_types.go

```diff
@@ -464,6 +507,25 @@ func (s SearchParamInfo) clone() SearchParamInfo {
 	return newInfo
 }
 
+func createReverseChainedQueryInfo(resource, modifier, value string) SearchParamInfo {
```

Comment:
It looks like `value` isn't used.  Should it be?  If not, remove it from the signature.